### DISCUSSION TITLE
Building an SNES cable

### DISCUSSION THREAD

**Discussion by vsilvalopes** [darthcloud/BlueRetro#331] [Q&A]
Hi everyone!

I'm planing to do a Blueretro cable, for SNES, But I have some questions about the schematic that @darthcloud published on the Github page.

I'm not very good understanding electronic schematics, so, here's my questions

- On the bill of materials, we have three 74AHCT125N, I'm my understanding, we only need three, if wiring for multitap support right?

- On the Schematics for the SNES cable, I see three 74AHCT125N, identified by U1, U2 and U3. On the U1, the U1B part, is marked with a blue dotted box, and also U1D too. But U1D is marked as "Only for multitap support" this applies to U1B? 
Or in my case that I'm building the cable for two players only, I need to wire U1B too and discard the wiring of U1D?

- On the schematics, I see that the points P1-CLK, P1-SEL, LATCH, P2-CLK, P2-SEL have to be wired to the 5V point on the DB25, but those points are also wired to U2. My question is, do I have to wire those points to the 5v AND to their respective points on the 74AHCT125N?

- All the points marqued as GND, I have to wire to the GND point on the DB25 right?

Sorry if those questions seems to obvious.
I've already built successfully a cable for PS1/PS2, so besides not being an expert, I know how to solder and wire properly ! :D

Thanks in advance!

**Comment by darthcloud** [comment]
1. If building a 2 player cable only, you only need 2x 74AHCT125N .

2. Ignore everything U3x
   For U1B & U1D simply wire the pins 5 & 12 of U1 to GND instead of IO21 & IO25.

3. They are not connect to 5V, they are connected to a pull-up resistor (which is connect to 5V)
   This is for PAL system only, for US/JAP you can omit the resistors.

4. Yes wired All GND together.

**Reply by vsilvalopes** [reply]
Thanks for the reply @darthcloud !

Last question:

3- Ok, I understood not needing the resistors, as it IS only for PAL systems.
So, even without the resistors, I still need to wire P1-CLK, LATCH, P1-SEL, P2-CLK and P2-SEL to 5V?
Or I must follow the diagram, were :

- P1-CLK goes to 5 on U2
- P2-CLK goes to 9 on U2
- LATCH goes to 2 on U2

And I must ignore P1-SEL and P2-SEL as those are used for multitap suport only right?

Again, thanks for the help!

**Reply by darthcloud** [reply]
Dont wire any of those to 5V.

Ignore both SEL its for multitap.
